src/main.rs

```diff
@@ -112,6 +112,37 @@ impl TryFrom<&VerbosityFlags> for Option<Verbosity> {
     }
 }
 
+#[derive(Debug, StructOpt)]
+struct UnstableOptions {
+    /// Use the original manifest (Cargo.toml), do not modify for build optimizations
+    #[structopt(long = "unstable-options", short = "Z", number_of_values = 1)]
+    options: Vec<String>,
+}
```

Comment:
How would this look like if we specify multiple unstable options? Each having their own `-Z` or one `-Z` for all? I guess we should mimic `rustc` here that requires one `-Z` per unstable feature usage.

Reply:
Indeed that should work i.e. 1 `-Z` per option, I took this pattern from https://github.com/TeXitoi/structopt/blob/master/examples/keyvalue.rs#L22